app/src/main/java/edu/uc/reedws/musiclink/dto/PlaylistDTO.kt

```diff
@@ -0,0 +1,9 @@
+package edu.uc.reedws.musiclink.dto
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName="playlist")
```

Comment:
```suggestion
@Entity(tableName="playlists")
```
@devagrawal09 Unless theres some android reason for this, I prefer plural table names